tests/yam/agama/auto.pm

```diff
@@ -8,13 +8,29 @@ use base 'y2_installbase';
 use strict;
 use warnings;
 
-use testapi 'assert_screen';
-
+use testapi;
+use utils 'clear_console';
 
 sub run {
     assert_screen('agama-main-page', 120);
-    assert_screen('agama-installing', 60);
+    assert_screen('agama-installing', 120);
+
+    my ($ext) = get_required_var('AGAMA_AUTO') =~ /(\.[^.]+)$/;
```

Comment:
could you please remove this reboot in the .sh to see that behaves exaclty the same, so we don't have to do conditional logic:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/yam/agama/auto/alp.sh#LL8C1-L8C13

Reply:
done.